code/modules/mob/pulling.dm

```diff
@@ -15,7 +15,11 @@
 				return
 			stop_pulling()
 
-		src.pulling = AM
+		if(AM.pulledby)
```

Comment:
Ah shit I didn't test this before, I accidentally broke a pull by pulling someone who was pulling a restrained person
Note to self: Test this.